### DISCUSSION TITLE
E1 split deployment?

### DISCUSSION THREAD

**Discussion by khooi8913** [srsran/srsRAN_Project#811] [General Help] [answered]
Hi all,

Does srsRAN currently support the E1 split, like deploying the CU-CP and (multiple) CU-UP on different hosts?

If I understand correctly, from the configuration reference, it seems that the CU-CP and CU-UP are still monolithic, and E1 is only used internally?

If the E1 split is possible, could someone share a configuration?

Thanks!

**Comment by dvdgrgrtt** [comment, marked as answer]
Hi @khooi8913,

no, the split is not possible and, as you say, CU-CP and CU-UP are still monolithic. Sorry for the inconvenience.

**Reply by khooi8913** [reply]
Hi @dvdgrgrtt, thanks for the clarification.
Any plans for srsRAN to support CU-CP and CU-UP split in the future?

**Reply by dvdgrgrtt** [reply]
We are looking into it, but we don't have a release date yet.
